himbaechel/uarch/gowin/gowin.h

```diff
@@ -106,6 +110,7 @@ NPNR_PACKED_STRUCT(struct Tile_extra_data_POD {
     int32_t class_id;
     int16_t io16_x_off;
     int16_t io16_y_off;
+    int32_t i3c_capable;
```

Comment:
are there likely to be other flags like this you want to store here? instead of using a whole int32_t for this, what about an int32_t flags field with one bit for this and the rest reserved for future, similar, tile flags